### PR TITLE
Update README.md

### LINKED ISSUE #59: Add Dockerfile path to README
Add the path to `Dockerfile` in the README

### PR DESCRIPTION
**Pull request by khabdrick**
## Description
This pull request is just to help the user navigate to the path that contains the `Dockerfile` before running `docker build`. 

Fixes #59

### CODE REVIEW

**Comment by khabdrick** [comment]
Hello @chicken-biryani did I make this PR the right way?

**Comment by chicken-biryani** [comment]
Hey... @khabdrick the people, at moja global, we go to for documentation review work are- @HarshCasper and @sarahhaggarty.

So I'll add them in the reviewer list.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The project aims to build a continuous deployment pipeline to offer FLINT as a S
 
 In order to run the REST API, please follow the following steps: - 
 
-1. `docker build -t flint-api .`
+1. `cd local/rest_api_flint.example && docker build -t flint-api .`
 2. `docker run --rm -p 8080:8080 flint-api`
```
Comment (sarahhaggarty) [inline README.md:25]: Hi @khabdrick, I can see these are just terminal commands on their own. Can you help the reader out by explaining what they should enter these commands into (I'm assuming any terminal will do), and in which directory/folder? If you could do the same for lines 38-42 that would also be great!

**Comment by khabdrick** [comment]
@sarahhaggarty I have made the updates

**Review by sarahhaggarty — APPROVED** [review]

**Comment by chicken-biryani** [comment]
@khabdrick can you please resolve the conflicts?

**Comment by khabdrick** [comment]
@chicken-biryani someone else has merged the corrections, so this can be closed.

**Pull request closed without merging**